reinforcement_learning/examples/python/rl_sim.py

```diff
@@ -51,40 +50,45 @@ def __init__(self, args):
 
     def loop(self):
         round = 0
-        while (True):
+        stats = {person._id : [[0,0] for _ in self._actions] for person in self._people}
+        ctr = [0,0]
+        while round != self._options.num_rounds:
```

Comment:
Even though I am sure this would be fine because the variable is changed as the first thing in the loop, this would feel a tiny safer if it was less than:
```suggestion
        while round <= self._options.num_rounds:
```

Feel free to wontfix this!